webui/react/src/types.ts

```diff
@@ -204,6 +204,7 @@ export enum RunState {
   StoppingError = 'STOPPING_ERROR',
   Errored = 'ERROR',
   Deleted = 'DELETED',
+  Unspecified = 'UNSPECIFIED',
```

Comment:
question: Just curious, do we know when this case comes up? Or is this just a fall-back?